docker/serve/Dockerfile

```diff
@@ -3,7 +3,7 @@ ARG CUDA="10.1"
 ARG CUDNN="7"
 FROM pytorch/pytorch:${PYTORCH}-cuda${CUDA}-cudnn${CUDNN}-devel
 
-ARG MMCV="1.3.12"
+ARG MMCV="1.3.13"
```

Comment:
Should also update mmcv version [here](https://github.com/open-mmlab/mmsegmentation/blob/master/docker/Dockerfile#L4).